src/chain/sanitizers-impl.ts

```diff
@@ -34,6 +34,9 @@ export class SanitizersImpl<Chain> implements Sanitizers<Chain> {
   normalizeEmail(options?: Options.NormalizeEmailOptions) {
     return this.addStandardSanitization(validator.normalizeEmail, options);
   }
+  replace(new_value: any, values_to_replace: any[] = [null, undefined, NaN, '']) {
+    return this.customSanitizer(value => (values_to_replace.includes(value) ? new_value : value));
```

Comment:
uh... I was ready to complain about using `NaN` here, but seems like I would be mistaken:

```
gustavohenke ~ $ node
Welcome to Node.js v12.16.1.
Type ".help" for more information.
> [NaN].includes(NaN)
true
> NaN === NaN
false
```